server/server.go

```diff
@@ -27,7 +27,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/cockroachdb/c-snappy"
+	snappy "github.com/cockroachdb/c-snappy"
```

Comment:
For some reason my setup fails to find this and removes the c-snappy line unless I explicitly rename it to how it's accessed in the file itself. Not sure why, but I'll revert this.